db/schema.rb

```diff
@@ -56,7 +56,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "title",                    limit: 255
-    t.boolean  "accessible",               limit: 1
```

Comment:
We shouldn't have these changes. I wonder if maybe you ran the migrations on your host machine (as opposed to your VM) and it has a different MySQL version?